src/commands/sns_canisters_summary.rs

```diff
@@ -0,0 +1,39 @@
+use crate::lib::signing::{sign_ingress_with_request_status_query, IngressWithRequestId};
+use crate::lib::{AnyhowResult, TargetCanister};
+use crate::SnsCanisterIds;
+use candid::Encode;
+use clap::Parser;
+use ic_base_types::{CanisterId, PrincipalId};
+use std::str::FromStr;
+
+#[derive(Parser, Debug)]
```

Comment:
I didn't see a debug use of this command anywhere. Is this here for a special reason?